pkg/option/config.go

```diff
@@ -652,6 +652,10 @@ const (
 	// option.IPAM
 	IPAMAzure = "azure"
 
+	// IPAMOperator is the value to select the Operator IPAM mode for
+	// option.IPAM
+	IPAMOperator = "operator"
```

Comment:
The name is technically correct but probably not very meaningful to users. I suggest we use something like `cluster-pool` to indicate that this gives control to specify a cluster-wide pool of addresses.